operate/client/src/modules/components/SortableTable/index.tsx

```diff
@@ -166,41 +173,64 @@ const SortableTable: React.FC<Props> = ({
                   <tbody aria-live="polite" data-testid="data-list">
                     {rows.map((row) => {
                       const isSelected = checkIsRowSelected?.(row.id) ?? false;
+                      const operationErrorMessage =
+                        rowOperationError?.(row.id) || null;
+
+                      const {batchOperationId} =
+                        getProcessInstancesRequestFilters();
+
+                      const expandRowStyleClasses = () => {
+                        if (operationErrorMessage) {
+                          return 'errorRow';
+                        } else if (!operationErrorMessage && batchOperationId) {
+                          return 'successRow';
+                        } else {
+                          return '';
+                        }
+                      };
 
                       return (
-                        <TableRow
-                          {...getRowProps({row})}
-                          isSelected={isSelected}
-                          $isClickable={selectionType === 'row'}
-                          aria-selected={isSelected}
-                          onClick={() => {
-                            if (selectionType === 'row') {
-                              onSelect?.(row.id);
-                            }
-                          }}
-                        >
-                          {selectionType === 'checkbox' && (
-                            <TableSelectRow
-                              {...getSelectionProps({row})}
-                              checked={isSelected}
-                              onSelect={(event) => {
-                                getSelectionProps({row}).onSelect(event);
+                        <React.Fragment key={row.id}>
+                          <TableExpandRow
+                            className={expandRowStyleClasses()}
```

Comment:
@pedesen I agree with the consistency point, however when I started working on this issue I did try to use a custom prop first and I don't think this can be done here. 

For the prop to work it would have to be on the row, but many of the styles I implemented don't actually work when I try applying them directly to the element (even with !important flags).

That's the reason why some of the styles are on the common parent component (the actual table component). This is most likely due to the fact that the CDS applies **a lot** of styles, so to override them (due to CSS specificity rules) this is the only way I found to make it work.

Do you think this is a blocker? I'm open to suggestions if you have any ideas regarding this.

Reply:
No, I think it's not a blocker. I don't see another way to make this work, so let's keep it as it is.